api/camunda/camunda-openapi.yaml

```diff
@@ -1281,15 +1444,54 @@ components:
     ProcessInstanceFilterRequest:
       type: object
       properties:
-        key:
-          type: array
-          items:
-            type: integer
-            format: int64
-        variables:
+        running:
+          type: boolean
+        active:
+          type: boolean
+        incidents:
+          type: boolean
+        finished:
+          type: boolean
+        completed:
+          type: boolean
+        canceled:
+          type: boolean
+        retriesLeft:
+          type: boolean
+        errorMessage:
+          type: string
+        activityId:
+          type: string
+        startDate:
+          type: string
+          format: date-time
+        endDate:
+          type: string
+          format: date-time
+        bpmnProcessId:
+          type: string
+        processDefinitionVersion:
+          type: integer
+          format: int32
+        variable:
+          allOf:
+            - $ref: "#/components/schemas/ProcessInstanceVariableFilterRequest"
+        batchOperationId:
+          type: string
+        parentProcessInstanceKey:
+          type: integer
+          format: int64
+        tenantId:
+          type: string
+    ProcessInstanceVariableFilterRequest:
+      type: object
+      properties:
+        name:
+          type: string
+        values:
           type: array
           items:
-            $ref: "#/components/schemas/VariableValueFilterRequest"
+            type: string
```

Comment:
category: missing descriptions

owner: API author

Reply:
@mihail-ca -- in [a recent PR](https://github.com/camunda/camunda/pull/21552) to the C8 REST API, you added or updated the following in the OpenAPI spec, each of which is missing `description` properties:

- [ProcessInstanceFilterRequest](https://github.com/camunda/camunda/blob/a5f191a60de81d929eec492189fd611537df8799/zeebe/gateway-protocol/src/main/proto/rest-api.yaml#L1486-L1527) schema
- [ProcessInstanceVariableFilterRequest](https://github.com/camunda/camunda/blob/a5f191a60de81d929eec492189fd611537df8799/zeebe/gateway-protocol/src/main/proto/rest-api.yaml#L1528-L1536) schema
- [ProcessInstanceItem](https://github.com/camunda/camunda/blob/a5f191a60de81d929eec492189fd611537df8799/zeebe/gateway-protocol/src/main/proto/rest-api.yaml#L1546-L1598) schema
- [OperationItem](https://github.com/camunda/camunda/blob/main/zeebe/gateway-protocol/src/main/proto/rest-api.yaml#L1711-L1743) schema

The descriptions are important because the docs ingest the spec from the `camunda` repo, and generate our API docs from it. I'm tagging you here in the `camunda-docs` repo, instead of the `camunda` repo, because that's where we (DevEx) noticed the descriptions missing.

Please add descriptions to these items, in the `camunda` repo's [C8 REST OpenAPI spec](https://github.com/camunda/camunda/blob/main/zeebe/gateway-protocol/src/main/proto/rest-api.yaml). Your changes will eventually propagate downstream to the docs here.

For guidance on writing the descriptions, [see here](https://github.com/camunda/camunda/blob/main/docs/rest-controller.md#openapi-extension). If you have any questions, feel free to ask. 

Thank you!